recipes/mingw-w64/config.yml

```diff
@@ -2,5 +2,5 @@ versions:
     "8.1":
       folder: "all"
     "8.0.2":
-      folder: "all"
+      folder: "linux"
```

Comment:
This confuses me

Reply:
I am not sure what confuses you, so what follows may not clear up anything, but I'll try anyway: The `linux` folder contains a recipe which is able to build MinGW toolchain on Linux. This recipe compiles a single version for now.
Mingw is a toolchain which always targets Windows (afaik) but can be compiled either om Linux or on windows.

Reply:
That's the magic piece I was missing.

You can target windows from linux!